### DISCUSSION TITLE
Changes Coming to Quire: Share Your Feedback!

### DISCUSSION THREAD

**Discussion by Erin-Cecele** [thegetty/quire#314] [FYI]
![markus-spiske-kK7uPfb2YWU-unsplash](https://user-images.githubusercontent.com/62075223/167968605-45940b1a-b60a-4e00-8bab-ebb469dac577.jpg)

**Exciting Development News** 

In advance of our upcoming open-source launch, we have been undertaking significant development work to improve Quire’s functionality. As part of this work, we are transitioning to a new JavaScript-based static site generator called **[Eleventy (11ty)](https://www.11ty.dev/)**. This move will allow for improved file structure, more advanced features, and better development support and contribution opportunities. Read our **[FAQ page ](https://quire.getty.edu/about/faq/)** for more information.

**We’d love to hear your thoughts. Please post your feedback and questions about technical considerations and/or what these changes mean for the future of Quire and your use of the tool.**

**Comment by antoinentl** [comment]
Great choice to switch to 11ty, _bravo_ for the transition work!

I have a remark about this decision: we agree that 11ty will simplify the installation of Quire, and the creation and the management of the templates (and other stuffs), but the dependencies will increase. One of the advantages of Hugo is that it can be installed via a single file.

And I have a question about the incoming development: is a support for structured bibliographies on the way? For examples for managing bibliographic data in JSON or BibTeX files (from Zotero). I already tried [markdown-it-biblatex](https://github.com/arothuis/markdown-it-biblatex/) and it works well!
